backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -163,7 +163,10 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
                 "Contract in Place" => GetWorkflow("SPL"),
                 "On Market" => GetWorkflow("SPL"),
                 "Pre-Market" => GetWorkflow("SPL"),
-                _ => _adminService.Workflow.GetForStatus(model.Status).OrderBy(w => w.SortOrder).Last()
+                "Not In Spl" => GetWorkflow("SPL"),
```

Comment:
Make sure importing project assigned the **ERP** workflow for **Not in SPL** status